src/sentry/static/sentry/app/components/modals/sentryAppPublishRequestModal.tsx

```diff
@@ -141,7 +139,7 @@ export default class SentryAppPublishRequestModal extends React.Component<Props>
             submitLabel={t('Request Publication')}
             onCancel={() => this.props.closeModal()}
           >
-            <JsonForm location={this.props.location} forms={forms} />
```

Comment:
TS caught that we weren't actually setting the prop value and it turned out to be unnecessary